### PR TITLE
Work Section

### PR DESCRIPTION
**Pull request by mnekx**
## My Portfolio Site: Milestone 2
In this, I have created
-the work section
-utilities.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -101,6 +101,88 @@ <h5>Skills</h5>
           >
         </div>
       </header>
+      <section>
+        <h1>MY RECENT WORK</h1>
+        <div class="seperator"></div>
+        <div class="cards">
+          <div class="card">
```
Comment (ReshyResh) [inline index.html:117]: - [ ] Good job introducing the works section with a `<section>` tag! However, as per requirement, the list of works is supposed to be positioned using the CSS property of `display:grid`, please make sure to change the CSS accordingly.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @mnekx , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] You used all the correct semantic tags, but you haven't styled them to match the Figma design. Please make sure to use, as per requirement, `display:flex` to display the content inside of each card, and use CSS properties such as margin, padding, text-color and others to recreate the design as close as possible!
Here's the difference.

 

- [ ] Please make sure to also add the 'About me' Section, not only the 'Works' one.

- [ ] Check under the review for more changes required.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mnekx) [reply index.html:117]: Hi,
Thanks for review. But, I do not know what happened but locally it loads nearly equal to the design. take a look at the screen shot below

image

Could you help me trouble shoot why!

Comment (mnekx) [reply index.html:117]: the .cards class has a display: grid

Comment (ReshyResh) [reply index.html:117]: Hi @mnekx, are you sure you pushed those changes? 
For the current pull request, you're requesting to merge the branch `work_sect` into `main`. 
If I take a look at this branch's code, the website looks like in the screenshot that I showed you.
Maybe you made changes on another branch and forgot to push them to this one?
Take a look at the [files changed](https://github.com/mnekx/my-portfolio/pull/5/files) section of this pull request and you'll see that your CSS doesn't have any display:grid or display:flex in it.

**Review by mmsesay — APPROVED** [review]
Hi @mnekx , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Please consider using the correct fonts, font-sizing as described in your chosen template.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**